README.md

```diff
@@ -0,0 +1,61 @@
+# Tesoro Installer of Packages
+
+**TIP** (Tesoro Installer of Packages) is a package manager built around PIP. It allows to keep only one copy of a
+package and to use it in different environments.
+
+## Installation
+
+Install this package using `pip install`. It will create [TIP Directory](#glossary) `~/tip`, but if you want to override
```

Comment:
Although conda and miniconda are installed in visible directories, it doesn't look convenient. User usually never touches these directories and they just steal user's attention. So I propose to install tip at `~/.tip` like is done in most of such software.